__tests__/endpoints.test.js

```diff
@@ -89,40 +89,24 @@ describe("'/api/articles' endpoint", () => {
                 expect(body.msg).toBe('Article not found')
             })
         })
-        test("GET /api/articles/:article_id returns 400 if NaN passed as an ID", () => {
+        test("GET /api/articles/:article_id returns 400 if invalid datatype passed as an ID", () => {
             return request(app)
             .get('/api/articles/ruairi')
             .expect(400)
             .then(({body}) => {
-                expect(body.msg).toBe('Article ID submitted is Not-a-Number (NaN)')
+                expect(body.msg).toBe('Article ID submitted is invalid datatype')
             })
         })
     })
     describe("GET all data from '/api/articles' endpoint", () => {
-        test("GET /api/articles returns an array with a length of 13", () => {
-            return request(app)
-            .get('/api/articles')
-            .expect(200)
-            .then(({body}) => {
-                expect(body.articles).toHaveLength(13)
-            })
-        })
-        test("GET /api/articles returns an array of objects which do not have a 'body' property", () => {
-            return request(app)
-            .get('/api/articles')
-            .expect(200)
-            .then(({body}) => {
-                body.articles.forEach(article => {
-                    expect(Object.keys(article)).not.toContain('body')
-                });
-            })
-        })
-        test("GET /api/articles returns an array of objects which contain specific properties", () => {
+        test("GET /api/articles returns an array of objecys with a length of 13, each object contains specific properties, excluding 'body'", () => {
```

Comment:
I'm guessing you refactored this based on some other feedback, but I'm going to make a further suggestion :) The test descriptions need to provide detail on behaviour, rather than the specifics of the test. When a request is made to the endpoint in production, they probably aren't going to get 13 things back. So something like "returns all articles" would be more descriptive of the behaviour.